lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2054 @@
+{
```

Comment:
<p>Dát data jako argument? Např</p><pre class="ql-syntax" spellcheck="false">get_uk_crime_data(latitude, longitude, date_from, date_to)
</pre><p> </p><p>Bacha na </p><pre class="ql-syntax" spellcheck="false">sys.exit
</pre><p>to rozbije celý kernel / interpret :) V tomhle případě bych tu vyjímku vůbec nechytal když se s ní nedělá nic než výpis.</p><p> </p><p>Rozhodně vytvořit appended_data uvnitř funkce, ne vně - už druhé zavolání by mělo překvapivéě hodně výsledků ...</p><p> </p><p>A taky se zbytečně dokola dělá</p><pre class="ql-syntax" spellcheck="false">api_url = "https://data.police.uk/api/stops-street"
</pre><p> </p><p>A obecně bych si trochu pohrál s pojmenováním proměnných ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
<p>ok, vyhodim chytání</p>